### ISSUE TITLE
RN Packaged on Web

### ISSUE THREAD

**Issue by ahwelgemoed** [pavjacko/renative#60]
Hi 
I'm trying to build a small app for ios, Android and Web as POC but I cannot get RN packages working on Web (They work fine on ios and Android). I've been trying Native-Base and/or React-Native-Easy-Grid. (Is there something I'm missing?)

Thanks

Here is the Error I get:
```
Entrypoint bundle = bundle.js bundle.f0a091cd09b580a1e966.hot-update.js bundle.js.map bundle.f0a091cd09b580a1e966.hot-update.js.map
[./node_modules/react-native-easy-grid/Components/Col.js] 286 bytes {bundle} [built] [failed] [1 error]
[./node_modules/react-native-easy-grid/Components/Grid.js] 297 bytes {bundle} [built] [failed] [1 error]
[./node_modules/react-native-easy-grid/Components/Row.js] 305 bytes {bundle} [built] [failed] [1 error]
[./node_modules/react-native-easy-grid/index.js] 164 bytes {bundle} [built]
[./src/screenHome.js] 3.59 KiB {bundle} [built]
    + 725 hidden modules

ERROR in ./node_modules/react-native-easy-grid/Components/Col.js 30:8
Module parse failed: Unexpected token (30:8)
You may need an appropriate loader to handle this file type.
|     if(this.props.onPress){
|       return(
>         <TouchableOpacity onPress={this.props.onPress}
|         {...this.prepareRootProps()}>
|     <View
 @ ./node_modules/react-native-easy-grid/index.js 6:0-35 8:0-25
 @ ./src/screenHome.js
 @ ./src/app.js
 @ ./index.web.js
 @ multi (webpack)-dev-server/client?http://0.0.0.0:8080 (webpack)/hot/dev-server.js ./index.web.js

ERROR in ./node_modules/react-native-easy-grid/Components/Row.js 31:12
Module parse failed: Unexpected token (31:12)
You may need an appropriate loader to handle this file type.
|       if(this.props.onPress){
|         return(
>             <TouchableOpacity onPress={this.props.onPress}
|             {...this.prepareRootProps()}>
|           <View
 @ ./node_modules/react-native-easy-grid/index.js 4:0-35 8:0-25
 @ ./src/screenHome.js
 @ ./src/app.js
 @ ./index.web.js
 @ multi (webpack)-dev-server/client?http://0.0.0.0:8080 (webpack)/hot/dev-server.js ./index.web.js

ERROR in ./node_modules/react-native-easy-grid/Components/Grid.js 42:8
Module parse failed: Unexpected token (42:8)
You may need an appropriate loader to handle this file type.
|     if(this.props.onPress){
|       return(
>         <TouchableOpacity onPress={this.props.onPress}>
|     <View
|       ref={component => this._root = component}
 @ ./node_modules/react-native-easy-grid/index.js 5:0-37 8:0-25
 @ ./src/screenHome.js
 @ ./src/app.js
 @ ./index.web.js
 @ multi (webpack)-dev-server/client?http://0.0.0.0:8080 (webpack)/hot/dev-server.js ./index.web.js
```

**Comment by danielmarino24i** [comment]
Never tried those libraries, but problem there seems to be that they are not being processed by babel, so detects JSX as invalid.

If you eject the project you can open your webpack.config.js and add it to the include path.

```
const babelLoaderConfiguration = {
    test: /\.js$/,
    // Add every directory that needs to be compiled by Babel during the build.
    include: [
        path.resolve(appDirectory, 'src'),
        path.resolve(appDirectory, 'node_modules', 'react-native-easy-grid'), <-- HERE
    ],
    use: {
        loader: 'babel-loader',
        options: {
            babelrc: false,
            presets: [
                ['module:metro-react-native-babel-preset'],
            ],
            plugins: ['@babel/plugin-proposal-class-properties'],
        },
    },
};
```

**Comment by pavjacko** [comment]
@danielmarino24i / @CHaNGeTe good suggestion!

@ahwelgemoed fixed in `0.21.12`

**Comment by pavjacko** [comment]
@ahwelgemoed added support for `native-base` (https://github.com/GeekyAnts/NativeBase)  in 0.21.14. 

simply `rnv plugin add` and pick native-base from options and re-run project
